contracts/modules/Wallet/VestingEscrowWallet.sol

```diff
@@ -0,0 +1,462 @@
+
+    vestingTemplates[templateCount] = VestingTemplate(_totalAllocation, _vestingDuration, _vestingFrequency);
+
+    templateCount = templateCount == 0 ? 0 : templateCount +1;
+
+    emit AddTemplate(
+      templateCount,
+      _totalAllocation,
+      _vestingDuration,
+      _vestingFrequency);
+  }
+
+  /**
+  * @notice Initiate a vesting schedule for any number of employees or affiliates
+  * @param _target Address of the employee or the affiliate
+  * @param _totalAllocation Total number of tokens allocated for the target
+  * @param _vestingDuration Total duration of the vesting schedule
+  * @param _startDate Start date of the vesting schedule
+  * @param _vestingFrequency Frequency of release of tokens
+  */
+  function initiateVestingSchedule(
+    address[] _target,
+    uint256[] _totalAllocation,
+    uint256[] _vestingDuration,
+    uint256[] _startDate,
+    uint256[] _vestingFrequency
+  )
+    public
+    onlyOwner
+    withPerm(ISSUER)
+  {
+    require(_target.length == _totalAllocation.length &&
+            _target.length == _vestingDuration.length &&
+            _target.length == _startDate.length &&
+            _target.length == _vestingFrequency.length);
+
+    for (uint i = 0; i < _target.length; i++) {
+      initiateVestingScheduleIterate(_target[i],
+                                     _totalAllocation[i],
+                                     _vestingDuration[i],
+                                     _startDate[i],
+                                     _vestingFrequency[i]);
+    }
+  }
+
+  /**
+  * @notice Initiate a vesting schedule for any number of employees from a template
+  * @param _target Address of the employee or the affiliate
+  * @param _templateNumber Template number to use
+  * @param _startDate Start date of the vesting schedule
+  */
+  function initiateVestingScheduleFromTemplate(
+    address[] _target,
+    uint256   _templateNumber,
+    uint256   _startDate
+  )
+    public
+    onlyOwner
+    withPerm(ISSUER)
+  {
+    VestingTemplate memory _vestingTemplate = vestingTemplates[_templateNumber];
+
+    for (uint i = 0; i < _target.length; i++) {
+      require(_target[i] != address(0));
+      initiateVestingScheduleIterate(_target[i],
+                                     _vestingTemplate.totalAllocation,
+                                     _vestingTemplate.vestingDuration,
+                                     _startDate,
+                                     _vestingTemplate.vestingFrequency);
+    }
+  }
+
+  /**
+  * @notice Cancel a vesting schedule for an employee or affiliate
+  * @param _target Address of the employee or the affiliate
+  * @param _vestingScheduleIndex Index of the vesting schedule for the target
+  * @param _revokeVestingSchedule True if the issuer is reclaiming the tokens out of the contract
+  */
+  function cancelVestingSchedule(address _target, uint256 _vestingScheduleIndex, bool _revokeVestingSchedule)
```

Comment:
could you please change the name of the function instead of word `cancel` could you use some meaning full word that signifies the working of function. 
ex- `drop`, `remove` etc